python/perspective/perspective/include/perspective/python/numpy.h

```diff
@@ -29,7 +39,7 @@ namespace numpy {
     /**
      * NumpyLoader fast-tracks the loading of Numpy arrays into Perspective, utilizing memcpy whenever possible.
      */
-    class PERSPECTIVE_EXPORT NumpyLoader {
+    class PERSPECTIVE_BINDING_EXPORT NumpyLoader {
```

Comment:
@sc1f is a class necessary?

Reply:
it follows the structure set by arrow_loader, and allows us to abstract all the numpy-specific stuff since its' all state-dependent